lib/output/utils.js

```diff
@@ -9,6 +9,30 @@ function hasOwn(obj, prop) {
   return Object.prototype.hasOwnProperty.call(obj, prop);
 }
 
+const getOwnPropertyDescriptors = typeof Object.getOwnPropertyDescriptors === "function" ?
+  Object.getOwnPropertyDescriptors :
+  // Polyfill exists before we require Node.js v6.x
```

Comment:
Either the wording or the Node.js-version seems a bit confusing here - could this say "Polyfill needed until we require Node.js v8.x" instead?

Reply:
Not sure what I was thinking… that sounds perfect.